# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/pointerevents/pointerevent_pointerrawupdate_after_adoption_from_parent.https.html.ini
# Commit: f8a2a169b296
# Full Hash: f8a2a169b2969d29e04733ae27a56459e5359973
# Author: Alexandru Marc <amarc@mozila.com>
# Date: 2025-05-16 09:19:19
# Regressor Bug: 1550462
# File Overlap Count: 1
# Description:
#   Revert "Bug 1550462 - part 3: Enable `pointerrawupdate` event r=smaug" for causing mochitest failures @ test_pointerrawupdate_event_count.html
#   
#   This reverts commit eb27445e0352d6b4df540adb40e21f72fc052e80.
#   
#   Revert "Bug 1550462 - part 2: Make `PresShell::HandleEvent` dispatch preceding `pointerrawupdate` event r=smaug,dom-core,edgar"
# ==============================================================================

diff -r 8d25773fc82f -r f8a2a169b296 testing/web-platform/meta/pointerevents/pointerevent_pointerrawupdate_after_adoption_from_parent.https.html.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/pointerevents/pointerevent_pointerrawupdate_after_adoption_from_parent.https.html.ini	Fri May 16 09:30:09 2025 +0300
@@ -0,0 +1,3 @@
+[pointerevent_pointerrawupdate_after_adoption_from_parent.https.html]
+  [`pointerrawupdate` events should be fired after the element is moved from a child doc to the parent doc]
+    expected: FAIL